lms/envs/bok_choy.env.json

```diff
@@ -112,6 +112,7 @@
         "city": "hidden",
         "country": "required"
     },
+    "REGISTRATION_EXTENSION_FORM": "openedx.core.djangoapps.user_api.tests.test_helpers.TestCaseForm",
```

Comment:
This list is alphabetical, so `REGISTRATION_EXTENSION_FORM` should go before `REGISTRATION_EXTRA_FIELDS`.